paddle/fluid/pybind/pybind.cc

```diff
@@ -1443,7 +1447,7 @@ All parameter, weight, gradient are variables in Paddle.
       .def("run_prepared_ctx",
            [](Executor &self, ExecutorPrepareContext *ctx, Scope *scope,
               std::map<std::string, const LoDTensor *> *feed_targets,
-              std::map<std::string, LoDTensor *> *fetch_targets,
+              std::map<std::string, FetchType *> *fetch_targets,
```

Comment:
Find a test including `run_prepared_ctx`, please check this is OK and add into unit test

Reply:
This function had been tested in TEST(inference, nlp) of test_inference_nlp.cc

Reply:
Good